### ISSUE TITLE
"Module 'numpy' has no ... member" error with PyLint 1.8.3

### ISSUE THREAD

**Issue by amyspark** [PyCQA/pylint#1975]
### Steps to reproduce

1. Use `pylint`  from within Visual Studio Code (I'm using the Insiders build, 1.22.0-insider).

### Current behavior

This example triggers `E1101:Module 'numpy.random' has no 'RandomState' member`, even though the IntelliSense help can perfectly find the class:

```python
import numpy as np
import seaborn as sns
import matplotlib.pyplot as plt

sns.set(style="dark")
rs = np.random.RandomState(50)

# Set up the matplotlib figure
f, axes = plt.subplots(3, 3, figsize=(9, 9), sharex=True, sharey=True)

# Rotate the starting point around the cubehelix hue circle
for ax, s in zip(axes.flat, np.linspace(0, 3, 10)):

    # Create a cubehelix colormap to use with kdeplot
    cmap = sns.cubehelix_palette(start=s, light=1, as_cmap=True)

    # Generate and plot a random bivariate dataset
    x, y = rs.randn(2, 50)
    sns.kdeplot(x, y, cmap=cmap, shade=True, cut=5, ax=ax)
    ax.set(xlim=(-3, 3), ylim=(-3, 3))

f.tight_layout()
plt.show()
```

### Expected behavior

No errors with the given example.

### pylint --version output

No config file found, using default configuration
pylint 1.8.3,
astroid 1.6.2
Python 3.6.4 (default, Mar 22 2018, 14:11:17)
[GCC 4.2.1 Compatible Apple LLVM 9.0.0 (clang-900.0.39.2)]

**Comment by PCManticore** [comment]
Thanks for the report!

**Comment by PCManticore** [comment]
THis is fixed on master with the `--extension-pkg-whitelist=numpy` option.

**Comment by PCManticore** [comment]
That is with pylint==2.0 and astroid==2.0

**Comment by Trubroadband** [comment]
E1101:Module 'numpy.random' has no 'RandomState' member

**Comment by lai-bluejay** [comment]
If you don't want to add more config, please add this code to your config file, instead of 'whitelist'.


```json
{
"python.linting.pylintArgs": ["--generate-members"],
}
```
relevent question: [how-do-i-get-pylint-to-recognize-numpy-member](https://stackoverflow.com/questions/20553551/how-do-i-get-pylint-to-recognize-numpy-members/54980614#54980614)

**Comment by SebastianoX** [comment]
Though in this case there will be no error when importing a non-existing member of numpy.
For example 
```
import numpy as np

np.non_existing()  # no error here
```

**Comment by graingert** [comment]
> If you don't want to add more config, please add this code to your config file, instead of 'whitelist'.
> 
> ```json
> {
> "python.linting.pylintArgs": ["--generate-members"],
> }
> ```
> 
> relevent question: [how-do-i-get-pylint-to-recognize-numpy-member](https://stackoverflow.com/questions/20553551/how-do-i-get-pylint-to-recognize-numpy-members/54980614#54980614)

pylint: error: no such option: --generate-members

**Comment by graingert** [comment]
setting this in my `pyproject.toml` works for me:

```
 [tool.pylint.MASTER]
 load-plugins = "pylint_django"
+extension-pkg-whitelist = "numpy"
+
 
```
